packages/vue-generator/src/expressions/class-members/method.ts

```diff
@@ -10,13 +10,26 @@ export function compileMethod(
 ): string {
   return `${expression.name}(${
     expression.parameters
-  })${expression.body.toString(options)}`;
+  })${expression.body?.toString(options)}`;
 }
 
 export class Method extends BaseMethod {
+  compileBody(options?: toStringOptions): string {
```

Comment:
You can move "this.modifiers.indexOf('abstract')" to a variable to reduce code repeat